portfolio/src/main/java/com/google/sps/objects/User.java

```diff
@@ -0,0 +1,42 @@
+package com.google.sps.objects;
+
+import java.util.Objects;
+
+public final class User {
```

Comment:
It looks to me like this class actually represents a comment rather than a user. Consider renaming.